static/openapi/eds-spec.json

```diff
@@ -0,0 +1,399 @@
+    "description": "The PhotonIQ Global Event Delivery Service (EDS) offers a robust platform capable of streaming live event data directly to your applications. This system supports thousands of simultaneous connections, allowing for easily incorporating event-driven capabilities into your infrastructure. This ensures your applications receive updates in real-time, enhancing their reactivity and efficiency.\n\nFeatures:\n- Delivery of live event information through WebSockets, including a unique feature for event de-duplication.\n- Options to subscribe to specific events and publish new ones to the event stream\n- Comprehensive metrics on user connections.\n- A health check and versioning API for the service.\nFor detailed information on endpoints, how to format requests and responses, and guidelines for integrating EDS with your applications, please refer to the API documentation provided below.\n",
+    "version": "1.0.0"
+  },
+  "tags": [
+    {
+      "name": "Publish Events",
+      "description": "APIs related to event publishing."
+    },
+    {
+      "name": "Manage Subscriptions",
+      "description": "APIs related to event subscriptions and streaming."
+    },
+    {
+      "name": "Event Delivery Metrics",
+      "description": "APIs for retrieving metrics data related to EDS."
+    },
+    {
+      "name": "Event Delivery Status",
+      "description": "APIs for checking the health of EDS."
+    },
+    {
+      "name": "Event Delivery Version",
+      "description": "APIs for checking the version of EDS."
+    }
+  ],
+  "paths": {
+    "/api/es/v1/fabric/{fabricName}/stream/{streamName}/publish": {
+      "post": {
+        "summary": "Publish events",
+        "description": "Publish events to a specified stream asynchronously.",
+        "tags": [
+          "Publish Events"
+        ],
+        "parameters": [
+          {
+            "in": "path",
+            "name": "fabricName",
+            "required": true,
+            "schema": {
+              "type": "string"
+            },
+            "description": "The name of the fabric."
+          },
+          {
+            "in": "path",
+            "name": "streamName",
+            "required": true,
+            "schema": {
+              "type": "string"
+            },
+            "description": "The name of the stream."
+          },
+          {
+            "in": "query",
+            "name": "type",
+            "required": true,
+            "schema": {
+              "type": "string"
+            },
+            "description": "The type of the stream (local or global or collection)."
+          }
+        ],
+        "requestBody": {
+          "required": true,
+          "description": "The request body containing the event data to publish to the stream.",
+          "content": {
+            "application/json": {
+              "schema": {}
+            }
+          }
+        },
+        "responses": {
+          "200": {
```

Comment:
Based on this API, the 200 returns no response.  Should that then not return 204 (No Content)?

Reply:
Changed it to return 204 (in code as well).